### DISCUSSION TITLE
basic-extra-meal/pass-by-value

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#297] [Announcements]
# basic-extra-meal/pass-by-value

在 Java 中，参数传递采用值传递方式。对于原始数据类型，直接传递值，而对于对象，实际传递的是引用的值。这使得在方法调用时，无法直接修改原始参数，但可以更改引用指向的对象的属性。掌握 Java 参数传递机制有助于编写更加稳定可靠的程序。

https://javabetter.cn/basic-extra-meal/pass-by-value.html

**Comment by akenO8** [comment]
介绍值传递的时候，结尾说"Fortran 语言是通过引用传递的"。这里是不是有错误
